nodes/all_in_one_publisher.py

```diff
@@ -49,24 +49,27 @@ class ROSMessagePublisher:
     # sim time message initialization 
     pub_sim = rospy.Publisher('clock', Clock, queue_size=10)  
 
-    # odom messages initialization
-    pub_odom = rospy.Publisher('odom', Odometry, queue_size=10)
-
-    # laser scan messages initialization
-    pub_laser = rospy.Publisher('scan', LaserScan, queue_size=10)  
-
     # instantiate object of imu publisher
-    pub_imu = rospy.Publisher('imu', Imu, queue_size=10)  
+    pub_imu = rospy.Publisher('imu', Imu, queue_size=10)
 
     # instantiate object of tf publisher
-    pub_tf = rospy.Publisher('tf', TFMessage, queue_size=10)  
+    pub_tf = rospy.Publisher('tf', TFMessage, queue_size=10)
+
+    # instantiate object of float publisher
+    pub_float64 = rospy.Publisher('fs_output', Float64MultiArray, queue_size=10)
```

Comment:
what's this used for?

Reply:
This is not supposed to be pushed. It's my local change for other implements.
I've removed it.